components/apimgt/org.wso2.carbon.apimgt.rest.api.authenticator/src/main/java/org/wso2/carbon/apimgt/rest/api/authenticator/AuthenticatorAPI.java

```diff
@@ -319,6 +322,27 @@ public Response callback(@Context Request request, @PathParam("appName") String
                     log.debug("Set cookies for " + appName + " application.");
                 }
                 // Redirect to the store/apis page (redirect URL)
+                String uiServiceUrl;
+                //The first host in the list "allowedHosts" is the host of UI-Service
+                String uiServiceHost = APIMConfigurationService.getInstance().getApimConfigurations()
+                        .getEnvironmentConfigurations().getAllowedHosts().get(0);
+                if (StringUtils.isEmpty(uiServiceHost)) {
+                    if (log.isDebugEnabled()) {
+                        log.debug("The first string in the list " +
+                                "'wso2.carbon.apimgt:environmentConfigurations:allowedHosts' configuration is empty.");
+                        log.debug("Read UI Service from 'wso2.carbon.apimgt.application:apimBaseUrl' configuration.");
+                    }
+                    uiServiceUrl = ServiceReferenceHolder.getInstance().getAPIMAppConfiguration().getApimBaseUrl();
+                } else {
+                    if (log.isDebugEnabled()) {
+                        log.debug("The first string in the list " +
+                                "'wso2.carbon.apimgt:environmentConfigurations:allowedHosts' configuration" +
+                                " is not empty. value: " + uiServiceHost);
+                    }
+                    uiServiceUrl = HTTPS_PROTOCOL + PROTOCOL_SEPERATOR + uiServiceHost + URL_PATH_SEPERATOR;
+                    log.info("UI Service: " + uiServiceUrl);
```

Comment:
use  log.info("UI Service {}: ", uiServiceUrl); of logging style

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/4896/commits/7f72562a3f2a9dc2595f82daf9f6b95e0fae92e9